### DISCUSSION TITLE
In Windows, suppressing console suppresses stderr, stdout output to my windowed (PySide6-based) application

### DISCUSSION THREAD

**Discussion by adw1011** [pyinstaller/pyinstaller#7466] [PyInstaller]
I think this issue might have been solved, but I haven't been able to find the answer.  The application sends stderr and stdout to a PySide6 window, but these messages do not show up if Windows does not open a console window along with the application.

Thanks for helping.

**Comment by bwoodsend** [comment]
That is exactly what windowed mode means. It's up to you to redirect errors to either a log file or some kind of pop up dialog.

**Reply by adw1011** [reply]
Thanks.  I usually work on a Mac.  My question was really a python question so sorry for posting here.
